server/src/test/java/org/elasticsearch/search/aggregations/bucket/terms/SignificantTextAggregatorTests.java

```diff
@@ -80,7 +78,10 @@ protected AggregationBuilder createAggBuilderForTypeTest(MappedFieldType fieldTy
     protected List<ValuesSourceType> getSupportedValuesSourceTypes() {
         // TODO it is likely accidental that SigText supports anything other than Bytes, and then only text fields
         return List.of(CoreValuesSourceType.NUMERIC,
```

Comment:
These are a consequence of changing to checking for TextSearchInfo.NONE rather than an index analyzer.  Again, it is sort of accidental that they work, and it may not make any real sense, but they do...